frontend/src/components/logged_in/Input.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+function Input({ name, type, reqd, value="", onChange }) {
```

Comment:
I suppose I could be wrong, but for parameters to have a default value, is it not better to put it at the end so that you can just leave it out? Otherwise you would have to put a value, or I suppose have two commas in a row in order to set onChange without setting value.

Reply:
I don't care about you changing it or anything btw, just curious. Since it works better to just move on.

Reply:
Since this is a component, `value` will actually be a prop in the JSX (e.g. `<Input value="whatever" ... `), so order doesn't matter.